Expensify/App
# fix: Search Page Order

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/6627
$ https://github.com/Expensify/App/issues/5699

### Tests | QA Steps
1. Open Search Page.
2. Search for some user name that does not start with #.
3. Search results should be in the following order:
1. 1: 1 chat.
2. group Chat.
3. Room.

4. Order will change based on the resemblance between search value and search result. e.g. If the search value matches the room name, the room name would be above. etc.
---

  1. In an account that has at least 1 workspace
  2. Open search
  3. Search for #announce room
  4. Delete a character and write it again
  5. room name should not duplicate

### Tested On


### Screenshots


#### Web | Desktop


https://user-images.githubusercontent.com/24370807/148835203-1b98d6a0-704c-493a-b0fb-72e8e83cf149.mp4


#### Mobile Web

![image](https://user-images.githubusercontent.com/24370807/148849609-0f861805-a128-4e5b-b389-7542137cb566.png)


#### iOS


#### Android

![image](https://user-images.githubusercontent.com/24370807/148848793-5fb9e798-7f36-46bb-9c77-3083e5593420.png)
